test/constraint.custom.async.test.js

```diff
@@ -23,11 +24,14 @@ const customHeaderConstraint = {
   }
 }
 
-test('should derive async constraint', t => {
+test('should derive multiple async constraints', t => {
   t.plan(2)
 
-  const router = FindMyWay({ constraints: { requestedBy: customHeaderConstraint } })
-  router.on('GET', '/', { constraints: { requestedBy: 'node' } }, () => 'asyncHandler')
+  const customHeaderConstraint2 = rfdc(customHeaderConstraint)
+  customHeaderConstraint2.name = 'requestedBy2'
+
+  const router = FindMyWay({ constraints: { requestedBy: customHeaderConstraint, requestedBy2: customHeaderConstraint2 } })
```

Comment:
To evaluate this branch: https://github.com/delvedor/find-my-way/blob/c4ffd9e1264c0559294170ddc5fbac8861fd24d3/lib/constrainer.js#L135